src/movepick.h

```diff
@@ -99,8 +99,8 @@ enum StatsType {
 // ButterflyHistory records how often quiet moves have been successful or
 // unsuccessful during the current search, and is used for reduction and move
 // ordering decisions. It uses 2 tables (one for each color) indexed by
-// the move's from and to squares, see www.chessprogramming.org/Butterfly_Boards
-// (~11 elo)
+// the move's from and to squares, as well as whether the to square is attacked, 
```

Comment:
` as well as whether the to square is attacked, ` I dont get this part?

Reply:
Removed